frontend/src/libs/components/link/styles.module.scss

```diff
@@ -1,4 +1,3 @@
 .link {
-  color: inherit;
```

Comment:
Why do we need to remove it?

Reply:
Removed `color` from Link styles because there were difficulties with changing the font color for the link, because of this, the color of the logo was wrong.

Reply:
We need to change the color of the parent component instead of changing it directly

Reply:
I believe `color: inherit` takes a color for link from the upper parent styles

Reply:
Okay, you mean that with `color: inherit` font color applies only from parent element color styling - if so, then let's remove it and pass color directly through class for link element 👍

Reply:
Yes, that's what I mean